generate-migration.sh

```diff
@@ -0,0 +1,21 @@
+
+readonly migration_name="${1:?"FATAL: missing a parameter please provide the name of the migration"}"
+readonly migration_path="./importer/analytics-datastore-clickhouse/migrations"
+readonly migration_full_path="$migration_path/$(date +%s)-$migration_name.js"
```

Comment:
```suggestion
readonly migration_full_path
migration_full_path="$migration_path/$(date +%s)-$migration_name.js"
```

Shell check wanted this, hate seeing yellow in my files :sweat_smile: :